models/labels/balancer_v2/labels_balancer_v2_pools_arbitrum.sql

```diff
@@ -0,0 +1,128 @@
+{{config(alias='balancer_v2_pools_arbitrum',
+        post_hook='{{ expose_spells(\'["arbitrum"]\',
+                                    "sector",
+                                    "labels",
+                                    \'["balancerlabs"]\') }}')}}
+
+WITH pools AS (
+    SELECT pool_id,
+           zip.tokens AS token_address,
+           zip.weights / pow(10, 18) AS normalized_weight,
+           symbol,
+           pool_type
+    FROM (
+        SELECT c.poolId AS pool_id,
+               explode(arrays_zip(cc.tokens, cc.weights)) AS zip,
+               cc.symbol,
+               'WP' AS pool_type
+        FROM {{ source('balancer_v2_arbitrum', 'Vault_evt_PoolRegistered') }} c
+        INNER JOIN {{ source('balancer_v2_arbitrum', 'WeightedPoolFactory_call_create') }} cc
+        ON c.evt_tx_hash = cc.call_tx_hash
+    )
+
+    UNION ALL
+
+    SELECT pool_id, zip.tokens AS token_address,  zip.normalizedWeights/pow(10, 18) AS normalized_weight, symbol, pool_type
+    FROM (
+        SELECT c.poolId AS pool_id, explode(arrays_zip(cc.tokens, cc.normalizedWeights)) AS zip, cc.symbol, 'WP' AS pool_type
+        FROM {{ source('balancer_v2_arbitrum', 'Vault_evt_PoolRegistered') }} c
+        INNER JOIN {{ source('balancer_v2_arbitrum', 'WeightedPoolV2Factory_call_create') }} cc
```

Comment:
Please follow the source syntax and modify source tables used in the other files.

Reply:
Thanks for noticing that @Hosuke, just commited these changes.